content/_changelogs/10.5.0.md

```diff
@@ -0,0 +1,89 @@
+## 10.5.0
+
+_Released 8/15/2022_
+
+**Features:**
+
+- Display a 'flaky test' indicator in the Specs Explorer for specs that are
+  known to be flaky in the Dashboard. Addresses
+  [#22656](https://github.com/cypress-io/cypress/issues/22656) &
+  [#22657](https://github.com/cypress-io/cypress/issues/22657).
+- The project drop down is now populated when connecting a project with an
+  organization that only has one project. Addresses
+  [#22936](https://github.com/cypress-io/cypress/issues/22936).
+- Added new `testIsolation` configuration option to allow users to revert to
+  `legacy` mode when `experimentalSessionAndOrigin` is set to true. Read more
+  about [test isolation](/guides/core-concepts/test-isolation) in Cypress to
+  learn more. Addresses
+  [#22279](https://github.com/cypress-io/cypress/issues/22279).
+- React 18 projects will now be scaffolded with the `cypress/react18` mount
+  registration during component setup. Addresses
+  [#23033](https://github.com/cypress-io/cypress/issues/23033).
+- The list of Vue components in the project are now available for selection when
+  generating a new spec. Addresses
+  [#22836](https://github.com/cypress-io/cypress/issues/22836).
+- If the user logs in to the Dashboard from the app, but the current project
+  does not have a project ID, the user will now be able to enter the connect
+  project workflow directly. Addresses
+  [#23022](https://github.com/cypress-io/cypress/issues/23022)
+- Adds Angular component testing support, including Angular template support and
+  support for Angular standalone components. Addresses
+  [#23089](https://github.com/cypress-io/cypress/issues/23089) &
+  [#22819](https://github.com/cypress-io/cypress/issues/22819) &
+  [#23096](https://github.com/cypress-io/cypress/issues/23096) &
+  [#23000](https://github.com/cypress-io/cypress/issues/23000) &
+  [#22991](https://github.com/cypress-io/cypress/issues/22991) &
+  [#23125](https://github.com/cypress-io/cypress/issues/23125).
```

Comment:
Looks like `[#22819](https://github.com/cypress-io/cypress/issues/22819)` was the epic for this work.

Reply:
Are you saying to remove that from the list?

Reply:
We should be able to remove the list for the single issue

Reply:
Updated [43949dc](https://github.com/cypress-io/cypress-documentation/pull/4668/commits/43949dc619c69fddd2b3ce1922478de129e95ef5)